java/spark/src/test/scala/org/apache/spark/sql/util/LanceArrowUtilsSuite.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.util
+
+import com.lancedb.lance.spark.LanceConstant
+import org.apache.arrow.vector.types.pojo.ArrowType
+import org.apache.spark.SparkUnsupportedOperationException
+import org.apache.spark.sql.types._
+import org.scalatest.funsuite.AnyFunSuite
+
+import java.time.ZoneId
+
+/**
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: Copyright The Lance Authors
+
+ The following code is originally from https://github.com/apache/spark/blob/master/sql/catalyst/src/test/scala/org/apache/spark/sql/util/ArrowUtilsSuite.scala
+ and is licensed under the Apache license:
+
+ License: Apache License 2.0, Copyright 2014 and onwards The Apache Software Foundation.
+ https://github.com/apache/spark/blob/master/LICENSE
+
+ It has been modified by the Lance developers to fit the needs of the Lance project.
+ */
```

Comment:
Could you move this up? It's confusing with the other license above.

Reply:
I just put it in the apache license header.